packages/in_app_purchase/ios/Classes/InAppPurchasePlugin.m

```diff
@@ -103,6 +105,15 @@ - (void)canMakePayments:(FlutterResult)result {
   result([NSNumber numberWithBool:[SKPaymentQueue canMakePayments]]);
 }
 
+- (void)getPendingTransactions:(FlutterResult)result {
+  NSArray<SKPaymentTransaction *> *transactions = [self.paymentQueueHandler getPendingTransactions];
+  NSMutableArray *transactionMaps = [NSMutableArray new];
```

Comment:
Nit: Google tradition is to avoid using convenience class methods except for [NSURL initWithString], which raises exception when string is nil.

Reply:
done